.wvr-ud/static-assets/styles.css

```diff
@@ -148,4 +148,2292 @@ main.loading #preview {
 .feather-plus-circle,
 .open .feather-minus-circle {
   display: inline-block;
-}
\ No newline at end of file
+}
```

Comment:
This should be broken down into partial files and imported.

Reply:
This was artificially long due to a bug. That bug has been removed.